buildtools/win/redirect-cc/redirect-cc.cc

```diff
@@ -52,6 +52,20 @@ const std::wstring EscapeArg(const wchar_t* arg) {
   return str;
 }
 
+const std::wstring GetPythonExe() {
+  std::wifstream wif(
+      L"..\\..\\brave\\vendor\\depot_tools\\python_bin_reldir.txt");
+  if (!wif.is_open()) {
+    wprintf(L"REDIRECT-CC: unable to open python_bin_reldir.txt.\n");
+    return std::wstring();
+  }
+  std::wstring line;
+  std::getline(wif, line);
+  wif.close();
+  line = L"..\\..\\brave\\vendor\\depot_tools\\" + line + L"\\python.exe";
+  return line;
+}
```

Comment:
is this fast enough? we can do this via env var, this way we don't need to access the file on each redirect-cc launch.
I did a similar lookup for python3 in JS [here](https://github.com/brave/brave-core/pull/11969/files#diff-6149b8e49614376fc44e8cd42d0ce8661e754d14d3bb4df131c3de178e852b0dR87).